lib/themes/dosomething/paraneue_dosomething/includes/helpers.inc

```diff
@@ -360,12 +360,8 @@ function paraneue_dosomething_get_themed_reportback_item($data) {
     return NULL;
   }
 
-  if ($data->id && !$data->disable_reactions) {
-    $data->reaction_totals = [
-      'heart' => dosomething_kudos_get_total_for_file_by_term_name($data->id, 'heart'),
-    ];
-
-    $data->existing_kids = dosomething_kudos_get_kudos_for_user_by_file($data->id);
+  if (! dosomething_kudos_disable_reactions($data->campaign->id)) {
```

Comment:
Oh I added it in the template logic, I can do it here since that makes more sense